api/projects.ts

```diff
@@ -60,6 +60,9 @@ export async function uploadProject(
   platformVersion?: string,
   intermediateRepresentation?: unknown
 ): HubSpotPromise<UploadProjectResponse | UploadIRResponse> {
+  if (fs.statSync(projectFile).size > 100 * 1024 * 1024) {
+    throw new Error('File size is too large. The maximum file size is 100MB.');
+  }
```

Comment:
I know we would need to make a data driven decision on this and not just pick this value arbitrarily.  I think this would be a good method to make users aware of the size of their project.  This could also just be a warning log letting them know that they are exceeding the threshold and it may cause problems